lms/djangoapps/learner_home/serializers.py

```diff
@@ -333,15 +333,26 @@ def get_enrollmentUrl(self, instance):
 class RelatedProgramSerializer(serializers.Serializer):
     """Related programs information"""
 
-    bannerUrl = serializers.URLField()
-    estimatedNumberOfWeeks = serializers.IntegerField()
-    logoUrl = serializers.URLField()
-    numberOfCourses = serializers.IntegerField()
-    programType = serializers.CharField()
-    programUrl = serializers.URLField()
-    provider = serializers.CharField()
+    bannerUrl = serializers.URLField(source="banner_image.small.url")
+    logoUrl = serializers.SerializerMethodField()
+    numberOfCourses = serializers.SerializerMethodField()
+    programType = serializers.CharField(source="type")
+    programUrl = serializers.SerializerMethodField()
+    provider = serializers.SerializerMethodField()
     title = serializers.CharField()
 
+    def get_numberOfCourses(self, instance):
+        return len(instance['courses'])
+
+    def get_logoUrl(self, instance):
+        return instance['authoring_organizations'][0]['logo_image_url']
```

Comment:
Did we verify that `authoring_organizations` is the correct field to look at? I thought maybe it would be `partner`?

Reply:
I am not seeing partner field in test data.